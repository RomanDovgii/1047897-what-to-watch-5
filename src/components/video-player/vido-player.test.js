import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const TestComponent = `<div/>`;
const SecondTestComponent = `<video/>`;
const ThirdTestComponent = () => {
  return (
    <video/>
  );
};

it(
    `Should render correct VideoPlayer with given children`,
    () => {
      const tree = renderer
      .create(
          <VideoPlayer>
            <TestComponent/>
          </VideoPlayer>
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);

it(
    `Should render correct VideoPlayer with given children`,
    () => {
      const tree = renderer
      .create(
          <VideoPlayer>
            <SecondTestComponent/>
          </VideoPlayer>
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);

it(
    `Should render correct VideoPlayer with given children`,
    () => {
      const tree = renderer
      .create(
          <VideoPlayer>
            <ThirdTestComponent/>
          </VideoPlayer>
      )
      .toJSON();

      expect(tree).toMatchSnapshot();
    }
);
