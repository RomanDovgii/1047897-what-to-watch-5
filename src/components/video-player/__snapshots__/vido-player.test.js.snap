// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correct VideoPlayer with given children 1`] = `<<div/> />`;

exports[`Should render correct VideoPlayer with given children 2`] = `<<video/> />`;

exports[`Should render correct VideoPlayer with given children 3`] = `<video />`;
